spec/books_spec.rb

```diff
@@ -0,0 +1,11 @@
+require_relative '../books'
+
+describe Book do
+  context 'testing the Book class' do
+    it 'create a new book when book class is initialised' do
+      book = Book.new('title', 'author')
+      expect(book.title).to eq 'title'
+      expect(book.author).to eq 'author'
+    end
+  end
+end
```

Comment:
- [ Required ] Please consider adding unit tests for the following methods in the `Book` class. It is part of the project requirements that all methods in a class are tested.
  
  - `add_rental(personal, date)`